src/components/Footer/Links.jsx

```diff
@@ -21,6 +21,12 @@ export default () => (
       >
         <IconSteam style={{ width: 14, height: 14 }} />
       </IconButton>
+      {location.location === '/' &&
```

Comment:
Honestly, I don't mind just putting the attribution on every page if it makes things simpler :)